go.mod

```diff
@@ -3,16 +3,10 @@ module github.com/containerd/zfs
 go 1.16
 
 require (
-	github.com/Microsoft/hcsshim v0.8.14 // indirect
-	github.com/containerd/containerd v1.4.3
+	github.com/Microsoft/hcsshim v0.8.15 // indirect
+	github.com/containerd/containerd v1.5.0-beta.1
```

Comment:
.3?

Reply:
let's land the other one @mikebrow https://github.com/containerd/zfs/pull/36